docker/Dockerfile.dashboard

```diff
@@ -0,0 +1,53 @@
+FROM node:18-alpine AS base
```

Comment:
We use `node:20.2.0-bullseye-slim` in `Dockerfile.dashboard`, let's align and not have multiple versions

Reply:
sure